ClientLib/src/main/java/com/o3dr/services/android/lib/gcs/follow/FollowLocation.java

```diff
@@ -0,0 +1,119 @@
+package com.o3dr.services.android.lib.gcs.follow;
```

Comment:
@kellyschrock let's reuse the [android.location.Location](https://developer.android.com/reference/android/location/Location.html) class instead of creating a new one.